Kickstarter-iOS/Views/Controllers/LoginToutViewControllerTests.swift

```diff
@@ -33,6 +33,7 @@ internal final class LoginToutViewControllerTests: TestCase {
     let intent = LoginIntent.generic
     let controller = LoginToutViewController.configuredWith(loginIntent: intent)
 
-    XCTAssertNotNil(controller.view.subviews.first as? UIScrollView)
+    // Due to the new design, a background imageView is now added as subview before the scroll.
+    XCTAssertNotNil(controller.view.subviews[1] as? UIScrollView)
```

Comment:
Hmm what's this test for? Does this change mean that the scroll to top functionality won't work for this screen?

Reply:
Correct. The problem here is that in order to match the designs, the view hierarchy had to change, since now a imageView was added to stack before the scrollView. As an option, I tried adding the background image view to the scrollView instead, but the scrolling looks weird with the background image moving.

Reply:
Oh one sec. I think I found a solution.